packages/flutter_tools/lib/src/windows/migrations/version_migration.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../base/file_system.dart';
+import '../../base/project_migrator.dart';
+import '../../cmake_project.dart';
+
+const String _cmakeFileBefore =
+'# Apply the standard set of build settings. This can be removed for applications\n'
+'# that need different build settings.\n'
+'apply_standard_settings(\${BINARY_NAME})\n'
+'\n'
+'# Disable Windows macros that collide with C++ standard library functions.\n'
+'target_compile_definitions(\${BINARY_NAME} PRIVATE "NOMINMAX")\n';
```

Comment:
Here and below: It might be nice to use `'''` multiline string syntax for these.